apps/settings/js/modules/storage/default_media_volume.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * DefaultMediaVolume:
+ *   - DefaultMediaVolume is an Observable that wraps the platform
+ *     DeviceStorage object.
+ *   - DefaultMediaVolume is a singleton that you can easily use it to fetch
+ *     some shared data across different panels.
+ *   - It has some observable properties: currentVolume.
+ * DefaultMediaVolume only update state and does not involve in any UI logic.
+ *
+ * @module DefaultMediaVolume
+ */
+define(function(require) {
```

Comment:
The module only provides one property `currentVolume` and it depends on `DeviceStorages` at the same time. How about simply adding a `defaultVolume` on `DeviceStorage`?